menuflow/nodes/http_request.py

```diff
@@ -97,7 +106,12 @@ async def request(self, session: ClientSession) -> Tuple(int, str):
         if self.data:
             request_body["json"] = self._data
 
-        response = await session.request(self.method, self._url, **request_body)
+        response = await session.request(
+            self.method, self._url, **request_body, trace_request_ctx=self._context_params
+        )
+
+        if response.status == 401:
```

Comment:
You must create the other, which will print a log with the relevant information.